reference-implementation/to-upstream-wpts/writable-streams/aborting.js

```diff
@@ -43,5 +41,256 @@ promise_test(t => {
     assert_not_equals(writer.ready, readyPromise, 'the ready promise property should change');
     return promise_rejects(t, new TypeError(), writer.ready, 'the ready promise should reject with a TypeError');
   });
-
 }, 'Aborting a WritableStream should cause the writer\'s fulfilled ready promise to reset to a rejected one');
+
+function promise_fulfills(expectedValue, promise, msg) {
```

Comment:
This is not necessary. Inside your promise_test, just do

```js
return promise.then(value => {
  assert_equals(value, expectedValue, msg);
});
```

the test will fail if it returns a rejected promise.